cmd/neofs-cli/internal/common/exit.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// ErrAwaitTimeout is a predefined error variable that represents
+// the expiration of a polling interval while awaiting a certain condition.
+var ErrAwaitTimeout = errors.New("await timeout expired")
+
 // ExitOnErr prints error and exits with a code that matches
```

Comment:
current statements about the function become wrong, lets get rid of `sdk library` phrase and describe code mapping like
```
// ExitOnErr prints error and exits with a code depending on the error type
//
//	0 if nil
//	1 if [sdkstatus.ErrServerInternal] or untyped
//	...
```